alns/accept/tests/test_hill_climbing.py

```diff
@@ -1,30 +1,92 @@
 import numpy.random as rnd
-from numpy.testing import assert_
+from numpy.testing import assert_, assert_equal, assert_raises
+from pytest import mark
 
 from alns.accept import HillClimbing
-from alns.tests.states import Zero, One
+from alns.tests.states import Zero, One, Two
 
+"""
+Hill Climbing
+"""
 
-def test_accepts_better():
+
+def test_hc_accepts_better():
     """
     Tests if the hill climbing method accepts a better solution.
     """
     hill_climbing = HillClimbing()
     assert_(hill_climbing(rnd.RandomState(), One(), One(), Zero()))
 
 
-def test_rejects_worse():
+def test_hc_rejects_worse():
     """
     Tests if the hill climbing method accepts a worse solution.
     """
     hill_climbing = HillClimbing()
     assert_(not hill_climbing(rnd.RandomState(), Zero(), Zero(), One()))
 
 
-def test_accepts_equal():
+def test_hc_accepts_equal():
     """
     Tests if the hill climbing method accepts a solution that results in the
     same objective value.
     """
     hill_climbing = HillClimbing()
     assert_(hill_climbing(rnd.RandomState(), Zero(), Zero(), Zero()))
+
+
+"""
+Late Acceptance Hill Climbing
+"""
```

Comment:
Comment: I would add this to the test names (e.g. `def test_lhac_...()`). You already do that for regular hill climbing just before, so it seems more consistent.